source/_docs/wordpress-redis.md

```diff
@@ -31,6 +31,13 @@ This method will store object cache values persistently in Redis while preservin
  ```
  terminus wp <site>.<env> -- plugin install wp-redis
  ```
+ 
+ For site networks, you will need to specify the site URL by adding that to the command:
+ 
+  ```
```

Comment:
Yep. See [Terminus command usage](https://pantheon.io/docs/terminus/examples/#module,-theme,-and-plugin-updates). But you should also install Terminus and try it out to confirm.